pkg/manager/repo/handler.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"openpitrix.io/openpitrix/pkg/constants"
+	"openpitrix.io/openpitrix/pkg/db"
+	"openpitrix.io/openpitrix/pkg/logger"
+	"openpitrix.io/openpitrix/pkg/manager"
+	"openpitrix.io/openpitrix/pkg/models"
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/utils"
+	"openpitrix.io/openpitrix/pkg/utils/sender"
+	"fmt"
+)
+
+func (p *Server) getRepo(repoId string) (*models.Repo, error) {
+	repo := &models.Repo{}
+	err := p.db.
+		Select(models.RepoColumns...).
+		From(models.RepoTableName).
+		Where(db.Eq("repo_id", repoId)).
+		LoadOne(&repo)
+	if err != nil {
+		return nil, err
+	}
+	return repo, nil
+}
+
+func (p *Server) DescribeRepos(ctx context.Context, req *pb.DescribeReposRequest) (*pb.DescribeReposResponse, error) {
+	s := sender.GetSenderFromContext(ctx)
+	logger.Infof("Got sender: %+v", s)
+	logger.Debugf("Got req: %+v", req)
+	var repos []*models.Repo
+	offset := utils.GetOffsetFromRequest(req)
+	limit := utils.GetLimitFromRequest(req)
+
+	fmt.Println(manager.BuildFilterConditions(req, models.RepoColumns...))
```

Comment:
We don't use `fmt` to print log, please use `logger.Debugf`

Reply:
ok